var/spack/repos/builtin/packages/libcatalyst/package.py

```diff
@@ -36,3 +46,28 @@ def cmake_args(self):
         ]
 
         return args
+
+    def smoke_test(self):
```

Comment:
Is your intention to eventually re-use this method as a stand-alone test as well as a build-time test?

See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#checking-an-installation.

Reply:
FWIW. I am unable to build the package using `spack -v install --test=root`:

```
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: $SPACK_ROOT/lib/spack/env/gcc/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "Findcatalyst.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "catalyst",
  but CMake did not find one.

  Could not find a package configuration file provided by "catalyst"
  (requested version 2.0) with any of the following names:

    catalystConfig.cmake
    catalyst-config.cmake

  Add the installation prefix of "catalyst" to CMAKE_PREFIX_PATH or set
  "catalyst_DIR" to a directory containing one of the above files.  If
  "catalyst" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
```

Reply:
Ideally this should be usable as a stand-alone test, but requires installing the examples which is currently not implemented in the RC.

Reply:
One option is to save the examples off after `install` (see https://spack.readthedocs.io/en/latest/packaging_guide.html#saving-build-time-files) then build and run the tests when those files are copied to the test stage directory.

Reply:
> Ideally this should be usable as a stand-alone test, but requires installing the examples which is currently not implemented in the RC.

In that case, why have a separate `smoke_test` method?